jdaviz/configs/specviz/plugins/line_analysis/line_analysis.py

```diff
@@ -23,8 +22,9 @@ class LineAnalysis(TemplateMixin):
     dialog = Bool(False).tag(sync=True)
     template_file = __file__, "line_analysis.vue"
     dc_items = List([]).tag(sync=True)
-    temp_function = Unicode().tag(sync=True)
-    available_functions = List(list(FUNCTIONS.keys())).tag(sync=True)
+    spectral_subset_items = List(["None"]).tag(sync=True)
```

Comment:
other plugins have (recently) been changed to be "Entire Spectrum" instead of "None", so probably should match that here (will also need update in `line_analysis.py` line ~74 and ~92).

Reply:
Good call.